src/Resistor/ResistorTable.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { Table } from 'react-bootstrap';
+import { ResistorColorEntry as Color, Resistor } from 'puzzle-lib';
+import './ResistorTable.css';
+
+class ResistorTable extends React.Component {
+  public render() {
+    return (
+      <div className="ResistorTable">
+        <Table striped={true} responsive={true}>
+          <thead>
+            <tr>
+              <th>Color</th>
+              <th>Value</th>
+              <th>Multiplier</th>
+              <th>Tolerance (%)</th>
+            </tr>
+          </thead>
+          <tbody>
+            {this.renderRows(Resistor.colorTable)}
+          </tbody>
+        </Table>
+      </div>
+    );
+  }
+
+  private renderRows(data: Color[]) {
+    return data.map((value: Color) => {
+      // TODO: Integrate color background into the name cell
+      return (
+        <tr key={value.name}>
+          <td style={this.cellColorStyle(value)}>{value.name}</td>
+          <td>{value.getDisplayValue()}</td>
+          <td>{value.multiplier}</td>
+          <td>{value.toleranceInPercent}</td>
+        </tr>
+      );
+    });
+  }
+
+  private cellColorStyle(value: Color) {
+    return {
+      backgroundColor: value.colorCode,
+      color: this.contrastingColor(value.colorCode),
+    };
+  }
+
+  private contrastingColor(background: String) {

```

Comment:
I'd add this to a common class somewhere.  This will be helpful for other tools.